### ISSUE TITLE
Add coverage report to pull requests

### ISSUE THREAD

**Issue by mrocklin** [dask/dask#2218]
It would be convenient when reviewing to see which added lines are covered and uncovered.  Does anyone have ideas or interest in setting this up?

**Comment by TomAugspurger** [comment]
Pandas has this setup through [codecov](https://codecov.io/gh)

I can try copying over the settings for dask/dask if you want.

**Comment by mrocklin** [comment]
It seems like a decent idea to me.  We should probably wait a bit for
others that this would affect to weigh in as well.

**Comment by pentschev** [comment]
I would like to enable code coverage, specifically to help me verify that I'm covering as much as possible in the context of https://github.com/dask/dask/pull/4543. Since I'm not an dask member/admin, I did create a [new repo](https://github.com/pentschev/dask-test/pull/1) to test coverage. I noticed there was already a `.coveragerc`, and only a few simple changes were required to get a first report, certainly not perfect, but it's a start.

May I submit a PR to enable codecov, for us to start a discussion? For that I would need just that someone ensure codecov is allowed for the Dask project.

**Comment by pentschev** [comment]
Of course, for now I don't want to enforce anything, but at least some reports would be helpful.

**Comment by pentschev** [comment]
@mrocklin any thoughts here?

**Comment by mrocklin** [comment]
Code coverage seems like a good idea to me. cc @jakirkham

**Comment by pentschev** [comment]
I opened a PR, let's what happens. Specifically if we need to grant codecov permissions to Dask.

**Comment by pentschev** [comment]
I think this can now be closed.
